go/vt/vtctl/vtctl.go

```diff
@@ -307,6 +307,9 @@ var commands = []commandGroup{
 			{"ValidateKeyspace", commandValidateKeyspace,
 				"[-ping-tablets] <keyspace name>",
 				"Validates that all nodes reachable from the specified keyspace are consistent."},
+			{"Reshard", commandReshard,
+				"[-skip_schema_copy] <keyspace.workflow> <source_shards> <target_shards>",
+				"Start a Resharding process. Example: Reshard ks.workflow001 '0' '-80,80-'"},
 			{"SplitClone", commandSplitClone,
```

Comment:
are you not removing this command in this PR?

Reply:
It's better not to remove anything legacy right now. I'm thinking we should wait a couple of releases before removing these.